homeassistant/components/iaqualink/sensor.py

```diff
@@ -45,13 +49,28 @@ def native_unit_of_measurement(self) -> str | None:
     @property
     def native_value(self) -> int | float | None:
         """Return the state of the sensor."""
-        if self.dev.state == "":
+        if self.dev.state in ("", "absent"):
```

Comment:
If it is absent, why would this sensor even exists?

Reply:
I did not create this integration, so I don't quite understand how the iAqualink API works or why does the integration create devices for stuff that might not be there. As mentioned in the other comment, the user can disable the specific device on the integration and the spamming would stop

Reply:
@frenck  - What do you need for us to get this into a release? I've had to disable the integration since it was spamming the logs so much